Lib/traceback.py

```diff
@@ -206,9 +206,9 @@ def _safe_string(value, what, func=str):
 
 # --
 
-def print_exc(limit=None, file=None, chain=True):
+def print_exc(limit=None, file=None, chain=True, **kwargs):
```

Comment:
Why are we adding kwarg-mania here instead of having an explicit `colorize=False` kwarg?

Reply:
Because this is to make it future-proof compatible. This is the same logic as we did for `print_exception`.

Reply:
Also, colorization is not yet officially supported for all functions. It's only assumed to be configured through env. vars for now (see the note in https://docs.python.org/3.14/library/traceback.html#module-traceback) so that's why we don't want to expose the parameter publicly yet.